docs/changelog.txt

```diff
@@ -49,3 +49,5 @@ Template for new versions:
 
 ## Misc Improvements
 - `stonesense`: changed announcements to be right-aligned and limited it to only show the most recent 10 announcements.
+- `stonesense`: config files are now in ``configs/stonesense``in the stonesense repo and are in ``dfhack-configs/stonesense`` when installed.
```

Comment:
```suggestion
- `stonesense`: `init.txt` config file is now read from ``dfhack-configs/stonesense/init.txt``
```

Reply:
non-public-facing changes don't need to go in the changelog (unless they have serious consequences for external developers)